docs/_content/ontwikkelaars/algemeen/notificaties.md

```diff
@@ -100,3 +127,163 @@ Bronnen:
 * [Digilevering koppelvlakspecificatie](https://www.logius.nl/sites/default/files/public/bestanden/diensten/DigiLevering/Koppelvlakspecificatie.pdf)
 * [Digilevering documentatie](https://www.logius.nl/diensten/digilevering/documentatie)
 * [Regie- en Zaakservices specificaties](https://www.gemmaonline.nl/images/gemmaonline/c/cd/Koppelvlakspecificatie_Regie-zaak_services_v1.0.pdf)
+
+# API specificatie (concept)
+
+[Voorbeeld sequence diagram (in-depth)](http://sequencediagram.org/index.html#initialData=C4S2BsFMAIDkHtQDMQGNICdIDsBQuAHAQw1FRGO2GgC0AlAYUJLIqKrkRBVSNBgbwAtgXjYcwZqTRsOdIgCMFYALIBFKa0rUAIo3z0GAWgB8CZGj4gBw0eKoAuaAAUA8gGUAKtAD0RAiA+AG4AjD4KGGL2ADrYPj4ADADMAEwhRgBeRADWOPG45tyW-NCCImISpvJKqmpODFh8MMDwAaix8clpmTl5PgVcPFY25fbAADxGRtXKwOpOAJI6TiEpSbiGk0aFQyVldhJOABKens7QKQkhHT4AQpHY0ADmmJBPIADOwFiYACbQQkg1CWKzW+VwAF4IQtsGAQERwJ9htAguxoAAqdGGTHQIhIF4PX5Q-B6YxmQbFaylWwVRwuDzePwBYJhRRRSCAqg4G5ddJZXJxfo7SkjA5UKqKWbzaDuACuCg+qAwIAUMFJ0BaGtaaB5qT5vUFAwsvD2NLGWxmtScrgA0rhSVthSaqftacBjqdzpdroKAIIKdmc6hEdgvIQh7LAcFQmFwhFIkqox6Y0k4vEEsREiEGRimJ3I11jJxuLy+fyBULhB72HyrJI+bAU52QD43W6YNAAC0k+dNo0qJktczqLkiv1l6ABLY+RBeWugIOgdaNRWb1P7VAtkqt0FtG0YjqbBbNh2gJzOFwSCRuADV4OAWvAQL98kP1KYTKTiwyy8zK43jWGD4fDEYAkxebA2w7VBu3yIA)
+
+### Abonneren
+
+Een consumer abonneert zich op notificaties door de volgende request naar de
+NC te sturen:
+
+```http
+POST /api/v1/subscribers
+
+Authorization: Bearer abcdef1234
+Content-Type: application/json
+
+{
+    "callbackUrl": "https://ref.tst.vng.cloud/zrc/api/v1/callbacks",
+    "auth": "Bearer aef34gh...",
```

Comment:
Ik neem aan dat dit `auth` attribuut voor de callback is (zodat het NC rechten heeft om de callback URL aan te roepen)?

Ik twijfel of dit de juiste manier is maar dat er iets van authenticatie nodig is lijkt me evident.

Reply:
Correct.

Ook github laat je toe om een secret o.i.d. te configureren, zou ook met een random string als onderdeel van de URL kunnen (dus optioneel veld?)